presto-server/src/main/provisio/presto.xml

```diff
@@ -241,6 +241,7 @@
             <unpack />
         </artifact>
     </artifactSet>
+
```

Comment:
nit : Can this move as a part of separate commit.

Reply:
See https://github.com/prestosql/presto/pull/5401.